### ISSUE TITLE
Workflow 3 (USMLE)

### ISSUE THREAD

**Issue by karafecho** [ncats/translator-workflows#7]
**WORKFLOW 3 HAS BEEN PLACED ON HOLD INDEFINITELY, PER IMPLEMENTATION TEAM CONSENSUS, JANUARY 10, 2019. THE TEAM WILL MOVE FORWARD WITH [WORKFLOW 3B](https://github.com/ncats/translator-workflows/issues/54).**

**Workflow Overview**
This workflow was developed in an effort to take advantage of the fact that USMLE questions have 'ground truth' answers (correct and incorrect) that have been very well vetted. In addition, the workflow does not focus on existing Translator use cases (e.g., asthma, Fanconi, diabetes), and therefore, it broadens the scope of Translator efforts and helps to ensure that the prototype Translator system will not suffer from overfitting.

**Breakdown of USMLE Driving Question**

**USMLE question:** A 55 year-old woman with small cell carcinoma of the lung is admitted to the hospital to undergo chemotherapy. Six days after treatment is started, she develops a temperature of 38°C (100.4°F). Physical examination shows no other abnormalities. Laboratory studies show a leukocyte count of 100/mm3 (5% segmented neutrophils and 95% lymphocytes). Which of the following is the most appropriate pharmacotherapy to increase this patient’s leukocyte count?

- Correct answer (from USMLE Board): filgrastim
- Incorrect answers (from USMLE Board): darbepoetin; dexamethasone; interferon-alpha; interleukin-2; leucovorin

**USMLE question, key 'facts' (in italics):** A 55 year-old woman with *small cell carcinoma of the lung* is admitted to the hospital to undergo *chemotherapy*. Six days after treatment is started, she develops a temperature of 38°C (100.4°F). Physical examination shows no other abnormalities. Laboratory studies show a *leukocyte count of 100/mm3 (5% segmented neutrophils and 95% lymphocytes)*. Which of the following is the most appropriate pharmacotherapy to increase this patient’s leukocyte count?

- Note that USMLE questions are multiple choice, and the process of exclusion may require the use of additional information; this is something we will need to investigate 

**Specific USMLE question:** What drugs might ameliorate [leukopenia] in patients with [small cell carcinoma of the lung] receiving [chemotherapy]?

- Could be interpreted more specifically as [neutropenia]

**High-level general question:** What drugs might ameliorate adverse event [x] in patients with disease/condition [y] receiving treatment [z]?

**High-level Overview of Workflow**

*Module 1: Delineation of Symptom/Phenotype Polarity*

- Polarity refers to symptoms/phenotypes that might be considered to be opposites (e.g., high vs low, absent vs present). Greek/Latin prefixes/suffixes/roots are useful in this regard and great for training the Translator system (e.g., NLP), and developing ontologies, algorithms, etc.

- Examples of adverse events related to [leukopenia]: neutropenia, granulocytopenia, thrombocytopenia, pancytopenia

- Examples of adverse events not related (or opposite) to [leukopenia]: leukocytosis, neutrophilic leukocytosis, granulocytosis, thrombocytosis, pancytosis

*Module 2: Identification of Genes Associated +/- with Symptom/Phenotype*

*Module 3: Identification of Chemicals Associated +/- with Symptom/Phenotype*

*Module 4: Merger of Lists of Annotated Chemicals*

*Module 5: Exclusion/Enrichment/Classification of Chemicals based on Annotation or Activity*

*Module 6: Exclusion/Enrichment/Classification of Chemicals based on US FDA Approval Status, for Drug Repurposing and Discovery*

*Challenge Module X: Translation of Natural-language Question to Machine Question*

**Comment**
This ticket is for discussion/issues related to Workflow 3 as a whole. Separate tickets have been created for issues specific to particular Modules. The Bid Matrix for this Workflow can be found [here](https://docs.google.com/spreadsheets/d/1LR0-Wf_iQRxOaIrV06sawxLBIA9nRPnSvmD_30Ew-1Y/edit?usp=sharing).

**Contact**
Contact for Workflow 3 is Kara Fecho.
